extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/config/DialectVersions.java

```diff
@@ -0,0 +1,47 @@
+package io.quarkus.hibernate.orm.runtime.config;
+
+import org.hibernate.dialect.DatabaseVersion;
+
+/**
+ * Constants exposing the default versions of various DBs in Quarkus.
+ * <p>
+ * If, one day, Hibernate ORM's defaults catch up with all our default versions,
+ * we could consider relying on ORM's defaults going forward.
+ * <p>
+ * For H2, we will probably have to keep a constant here forever,
+ * as H2 is embedded and thus Quarkus determines its default version through the BOM.
+ * See https://github.com/quarkusio/quarkus/issues/1886
+ *
+ * @author Emmanuel Bernard emmanuel@hibernate.org
```

Comment:
?

Reply:
Ah right. That's because I originally renamed the `QuarkusH2Dialect` class. Will fix.